WcaOnRails/app/views/static_pages/about.html.erb

```diff
@@ -7,6 +7,9 @@
     <hr class="my-4">
 
     <a class="btn btn-primary btn-lg" href="https://www.nfggive.com/guidestar/82-3825954" role="button"><%= t('about.donation_banner.donate_credit') %></a>
```

Comment:
Here and in `en.yml` we have this analogous URL to Guidestar, could you unify that to match what you did for Paypal?

Reply:
Sure thing, I've made the change and rebased the PR!

Reply:
Perfect, thank you!